apps/ui/src/views/Landing.vue

```diff
@@ -39,6 +39,11 @@ const SOCIALS = [
       <div class="space-y-2">
         <div class="eyebrow">Learn more</div>
         <div class="space-y-2">
+          <div>
+            <a href="/#/explore">
+              Explore page <IH-arrow-sm-right class="inline-block -rotate-45" />
+            </a>
+          </div>
```

Comment:
Unless we actually want it independently (@bonustrack) from this PR can we remove it?

I was testing on my own space so I had to navigate to my space via URL anyway
```
https://app.safe.global/apps/open?safe=oeth:0xa1919d8538F69412F2DD2A001Fd76E34307d3F79&appUrl=https%3A%2F%2Fbafybeielq6yozdxzk65irjrxutidakz74ja7dtw5txvu626y3cwzfgmasa.on.fleek.co%2F%23%2Fs%3A0cf5e.eth
```

Reply:
I can confirm we dont want this on the landing page

Reply:
Weirdly, the Safe URL is not working in dev mode. but yeah I've added it to do our tests. will remove